### ISSUE TITLE
Update Website screenshots

### ISSUE THREAD

**Issue by ZeroCoolGOS** [MattsShack/Plex-Movie-Poster-Display#95]
@MattsShack We should update the website as some of the screenshots are out of date.

I will go through the page in the next week or so and generate new images for you to link to.

**Comment by MattsShack** [comment]
Yes. I was just thinking about this.

I have a few other changes, clarifications, that also need to be made.

**Comment by ZeroCoolGOS** [comment]
Sounds good. Let me know if you need anything for it.

**Comment by MattsShack** [comment]
I have updated the website. I am not sure I am %100 happy with it yet, but it is a start. :)

**Comment by ZeroCoolGOS** [comment]
Sounds good. 

I will have a look. There is also some documentation I did in the docs folder for the installation. With some screen shots as well. 

I will also try and up date it with more.  

What things are you not liking?

**Comment by MattsShack** [comment]
I think it looks good. lots of good information that I should copy to the website. 
Do you think we should consolidate to a single location so it is easier to keep up to date? For example on the website or GitHub? 

-Matt

**Comment by ZeroCoolGOS** [comment]
Sorry just saw this update. Yes. I think we should consolidate into the git repo as it would always be the source of truth. Also that way we can contribute and fix issues with documentation.

**Comment by MattsShack** [comment]
I agree. From now on Git will the the 'source of truth'. I will update the website to direct users here once I review the documentation.

**Comment by ZeroCoolGOS** [comment]
@MattsShack 

Sounds good.  I have created a bunch of tickets that will so get fed back into the documentation. 

Side request: I don't know what your relation to MangaValk and d8ahazard is, but would it be possible to get added to the list of contributors to this project?  Thank you.

**Comment by MattsShack** [comment]
I see that :) I like it. 
I think we should add d8ahazard to the contributors.

Thank you again,

**Comment by ZeroCoolGOS** [comment]
d8ahazard is already a contributor.  I meant I was asking for me to be added like d8 and Mana lol.
